packages/superset-ui-demo/storybook/shared/components/ResizableChartDemo.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from '@superset-ui/style';
+import React, { useState, ReactNode } from 'react';
+import { DecoratorFunction } from '@storybook/addons';
+import ResizablePanel, { Size } from './ResizablePanel';
+
+export const SupersetBody = styled.div`
+  background: #f5f5f5;
+  padding: 16px;
+  min-height: 100%;
+
+  .panel {
+    margin-bottom: 0;
+  }
+`;
+
+export default function ResizableChartDemo({
+  children,
+  panelPadding = 30,
+  initialSize = { width: 500, height: 300 },
```

Comment:
I was using initialSize 400x400 (1:1) so we can take screenshots for thumbnails easily.